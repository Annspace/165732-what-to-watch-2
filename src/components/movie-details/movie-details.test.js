import React from 'react';
import renderer from 'react-test-renderer';
import MovieDetails from './movie-details.jsx';
import mockData from '../../mocks/films.js';

it(`renders correctly`, () => {
  const MovieDetailsComponent = renderer
    .create(<MovieDetails movie={mockData.movies[0]}/>);
  expect(MovieDetailsComponent).toMatchSnapshot();
});
