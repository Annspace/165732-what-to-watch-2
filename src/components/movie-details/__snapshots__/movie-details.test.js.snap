// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Wes Andreson
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        <div>
          Bill Murray
        </div>
        <div>
          Edward Norton
        </div>
        <div>
          Jude Law
        </div>
        <div>
          Willem Dafoe
        </div>
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        1:22:13
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        fantasy
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        2010
      </span>
    </p>
  </div>
</div>
`;
